src/styles/catalog.scss

```diff
@@ -0,0 +1,164 @@
+$size-card-item: 200px;
+$background-color-main: #fff;
+$cost-color: #060B35;
+$price-color: #616070;
+$button-color: #00ACDC;
+
+
+.card {
+  display: grid;
+  grid-template-columns: repeat(1, $size-card-item);
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 48px;
+
+  &__title {
+    color: $cost-color;
+    height: 36px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin: 40px 16px 0;
+  }
+
+  &__item {
+    width: $size-card-item;
+    box-sizing: border-box;
```

Comment:
it's better to set for each element on the page (use `*` selector)
note it's only property that allowed to be added in `*` selector

Reply:
I don't fully understand the logic, may I ask you to explain it in more detail?